### DISCUSSION TITLE
questions

### DISCUSSION THREAD

**Discussion by nsteinme** [sn-lab/MouseGoggles#19] [Q&A] [answered]
Wouldn't the precise position of the eyepiece relative to the eye make a big difference in the image seen by the mouse? If so, how do you achieve correct and repeatable positioning? And, it isn't clear from the images or text unless I missed it: is the eyepiece coming up close to the mouse's eye, or does it encircle the eye and rest on the skin? 

Any opportunity for obtaining a video of the eye position/pupil diameter while the mouse uses these? 

From the images it looks like the monocular one has a small body and long cable, but the binocular one has a big box close to the mouse - is there any way for the binocular one to also have long cables so that the Pi and other components can be out of the way?

**Comment by misaacson01** [comment, marked as answer]
Good questions! About eye positioning, one big reason why we settled on the optical design we used (a single Fresnel lens with the display at the lens' focal length) is because in this case the exact eye position matters as little as possible. With the display at infinity focus, all the rays coming out of the lens would be parallel no matter where the eye is placed, and the only effect of a poorly positioned eye is reduced visual field coverage. In reality it's not so perfect, but what we've found is that it does work as intended, you don't need to position the eyes that accurately to get similar levels of visual clarity as using no lens and a display 10-30 cm away (as other setups do). The only positioning protocol/advice we've given to anybody who uses this is something like 'position the eye as close the lens center as you can, preferably with the eye inside the eyepiece enclosure and the enclosure touching the skin/fur around the eye'. Getting the eye close to the lens ensures that as much of the FOV is covered as possible, and reduces light pollution. With this protocol it seems to work great, even with the fact that there is some variation in inter-eye distance with mice of different sizes that would make exact position impossible with our fixed case design. One last point about this is that for some very differently sized mice, we could be clipping the FOV a bit if the eyes are significantly closer or further apart, so we're soon going to release a version of the cases with adjustable inter-eye distance.
For a little more discussion on this topic, check out this [thread](https://github.com/sn-lab/MouseGoggles/issues/2).

For video of eye position/pupil diameter during, we're actively looking into this possibility right now, and I think there could be some other labs trying this as well!

For your last question, yes we do have a version of the binocular headset with the Raspberry Pi detached, check out Supplementary Figure 2 in our [preprint](https://doi.org/10.21203/rs.3.rs-3301474/v1) to see what that looks like. There aren't any specific assembly instructions for this headset yet, but the assembly is very similar to the larger headset, except that it uses a SPI cable and connectors commonly used for electrophysiology to detach the eyepieces from the Pi. It's a more expensive design since this cable and connectors are pricey, but we're exploring some other, simpler options right now as well.
